### PR TITLE
[infra] Refactored some sh files with python

### PR DESCRIPTION
**Pull request by gz83**
1.Move *.bat files to //win_scripts/old. These files have not been updated for a long time and are not convenient to use.

2.Rewrite some sh files with python, currently only for the parts applicable to Windows.

3.Some other related changes.

@Alex313031 

PTAL

### CODE REVIEW

**Comment by Alex313031** [comment]
@gz83 @hocheung-chromium Can you restore the WIN_CROSS_BUILD_INSTRUCTIONS.txt and WIN_INSTRUCTIONS.txt.

Also the BUILDING_WIN.md has some good changes, but also some ones I don't want and some stuff from upstream that I purposefully excluded.

I want to go through the file myself, but I don't want to lose your changes. 

I'm going to save your copy of the modified document, and then have you restore that file. Then I will use Meld and compare mine and your copy of it, and make changes as necessary, then commit it myself with credit at the top for your work.

So basically, restore those two .txt files and restore the BUILDING_WIN.md, and then I will merge this PR.

**Review by Alex313031 — APPROVED** [review]
Good
